docs/src/main/asciidoc/security-basic-authentication-tutorial.adoc

```diff
@@ -35,7 +35,8 @@ If you just want to examine the code, you can fast-forward to the completed exam
 * Download the {quickstarts-archive-url}[archive]
 * Clone the Git repository: `git clone {quickstarts-clone-url}`
 
-You can find the solution in the `security-jpa-quickstart` {quickstarts-tree-url}/security-jpa-quickstart[directory].
+You can find the solution using the `security-jpa` extension in the `security-jpa-quickstart` {quickstarts-tree-url}/security-jpa-quickstart[directory]
```

Comment:
I'm having some doubt here. Here we link to a reactive alternative but the the rest of the doc assumes a non-reactive tutorial is used, both tutorials are nearly identical but the dependencies/configuration differs, so the users preferring to work with the reactive equivalent would not be able to the current tutorial text one to one.

Can we discuss a couple of alternatives ? So 

Option 1: We just update this guide to use the reactive only Jakarta Perisistence. Do you see any side-effects on the users learning to use this IdentityProvider option ?
Option 2: In addition to adding a note here, add 2 more notes in a) section showing how to create an app, which extensions to use - the note would say - if you use the reactive one - start it like this... b) where it is shown how to configure - a similar note ?

May be you can split this PR in two ones ? We'll then merge your fix immediately and discuss for another few days how to present the reactive version ?

Thanks

Reply:
@michalvavrik See for example, https://quarkus.io/guides/qute#hello-world-with-jakarta-rest

Reply:
> Option 1: We just update this guide to use the reactive only Jakarta Perisistence. Do you see any side-effects on the users learning to use this IdentityProvider option ?

I think so far Hibernate, not Hibernate Reactive is user first choice, however it is possible to use both JPA datasource and reactive datasource together, therefore it would be for the best to allow users still work with Hibernate and document in this guide how to configure reactive datasource (Hibernate Reactive) for IdentityProvider. This way:

- users can still use Hibernate instead of Hibernate Reactive (AFAICT that's just general preference, not my personal choice)
- when authenticating reactive endpoints, we don't need to jump of IO thread, so it's better for performance

Problem with this is that ATM, both Security JPA and Security JPA Reactive are always using default datasource, so it will require some rework.

> Option 2: In addition to adding a note here, add 2 more notes in a) section showing how to create an app, which extensions to use - the note would say - if you use the reactive one - start it like this... b) where it is shown how to configure - a similar note ?

Yes. let's do that, thanks for suggestion, it will be better with this.

> See for example, https://quarkus.io/guides/qute#hello-world-with-jakarta-rest

+1